test-app/tests/integration/components/radio-test.gts

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable no-undef -- Until https://github.com/ember-cli/eslint-plugin-ember/issues/1747 is resolved... */
+/* eslint-disable simple-import-sort/imports,padding-line-between-statements,decorator-position/decorator-position -- Can't fix these manually, without --fix working in .gts */
+
+import { click, render, setupOnerror } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+
+import RadioControl from '@crowdstrike/ember-toucan-core/components/form/controls/radio';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+
+module('Integration | Component | Radio', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(<template>
+      {{! we do not require a label, but instead suggest using Field }}
+      {{! template-lint-disable require-input-label }}
+      <RadioControl @value="option" data-radio />
+    </template>);
+
+    assert.dom('[data-radio]').hasTagName('input');
+    assert.dom('[data-radio]').hasAttribute('type', 'radio');
+    assert.dom('[data-radio]').hasAttribute('value', 'option');
+
+    assert.dom('[data-radio]').isNotChecked();
+
+    assert.dom('[data-radio]').hasClass('bg-normal-idle');
+    assert.dom('[data-radio]').doesNotHaveClass('border-disabled');
+  });
+
+  test('it disables the radio using `@isDisabled`', async function (assert) {
+    await render(<template>
+      {{! we do not require a label, but instead suggest using Field }}
+      {{! template-lint-disable require-input-label }}
+      <RadioControl @value="option" @isDisabled={{true}} data-radio />
+    </template>);
+
+    assert.dom('[data-radio]').isDisabled();
+
+    assert.dom('[data-radio]').hasClass('border-disabled');
+    assert.dom('[data-radio]').hasClass('bg-transparent');
+    assert.dom('[data-radio]').doesNotHaveClass('bg-primary-idle');
+    assert.dom('[data-radio]').doesNotHaveClass('border-none');
```

Comment:
Suggestion: do you have to also had a check for not having `bg-disabled`?
https://github.com/CrowdStrike/ember-toucan-core/pull/69/files#diff-d84c4bf7f661538e00fe281442c5504a11ae3dc2ca5d55aa7ed07b3067972746R32

Reply:
Thank ya! Got this updated too!